app/dashboards/event_dashboard.rb

```diff
@@ -57,7 +59,7 @@ class EventDashboard < Administrate::BaseDashboard
   # Overwrite this method to customize how events are displayed
   # across all pages of the admin dashboard.
   #
-  # def display_resource(event)
-  #   "Event ##{event.id}"
-  # end
+  def display_resource(event)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)